data/json/items/comestibles/protein.json

```diff
@@ -0,0 +1,98 @@
+[
+  {
+    "id" : "hflesh_drink",
+    "copy-from": "protein_drink",
+    "type" : "COMESTIBLE",
+    "name" : "soylent green",
+    "name_plural" : "soylent green drinks",
+    "description": "A thin slurry of refined human protein mixed with water.  While quite nutritious, it is not particularly tasty.",
+    "material" : "hflesh"
+  },
+  {
+    "id": "hflesh_powder",
+    "copy-from": "protein_powder",
+    "type": "COMESTIBLE",
+    "name": "soylent green powder",
+    "name_plural": "servings of soylent green powder",
+    "description": "Raw, refined protein made out of people!  While quite nutritious, it is difficult to enjoy in its pure form.",
+    "material": [ "powder", "hflesh" ]
+  },
+  {
+    "id": "hflesh_shake",
+    "copy-from": "protein_shake",
+    "type": "COMESTIBLE",
+    "name": "soylent green shake",
+    "description": "A thick and tasty beverage made from pure refined human protein and nutritious fruit.",
+    "material": [ "fruit", "hflesh" ]
+  },
+  {
+    "id": "hflesh_shake_fortified",
+    "copy-from": "protein_shake_fortified",
+    "type": "COMESTIBLE",
+    "name": "fortified soylent green shake",
+    "description": "A thick and tasty beverage made from pure refined human protein and nutritious fruit.  It has been supplemented with extra vitamins and minerals",
+    "material": [ "fruit", "hflesh" ]    
+  },
+  {
+    "id": "protein_drink",
+    "type": "COMESTIBLE",
+    "comestible_type": "DRINK",
+    "name": "protein drink",
+    "description": "A thin slurry of refined protein mixed with water.  While quite nutritious, it is not particularly tasty.",
+    "weight": 288,
+    "volume": 1,
+    "price": 200,
+    "symbol": "~",
+    "color": "light_gray",
+    "phase": "liquid",
+    "container": "bottle_plastic",
+    "calories": 400,
```

Comment:
Lowering those calories to 100 would be better than leaving it as a "TODO: fix" kind of thing that can easily be forgotten until calories are recalculated for everything.